frontend/src/Components/ReviewsModal/ReviewsModal.css

```diff
@@ -0,0 +1,59 @@
+.modal {
+  display: block;
+  position: fixed;
+  z-index: 1;
+  padding-top: 60px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+.modal-content {
+  background-color: #fefefe;
+  margin: 5% auto;
+  padding: 20px;
+  border: 1px solid #888;
+  width: 80%;
+  max-width: 600px;
+  border-radius: 10px;
```

Comment:
Alternatively, the modal have a max-height so that it can't be larger than the viewport, and be made scrollable using `overflow: auto`.  However, I don't think one UX is strictly better than the other.

Reply:
I tried your suggestion and I think I like it better so will be keeping it.